internal/build/query_builder.go

```diff
@@ -112,7 +123,19 @@ func BuildSelectQuery(
 	groupBy []interface{},
 	join []JoinField,
 ) string {
-	ds := goqu.From(table)
+	var d goqu.DialectWrapper
+	switch dialect {
```

Comment:
why do you use switch-case here? You can use dialect parameter as input of goqu.Dialect.